test/unit/models/ministerial_role/minsterial_role_reindexing_concern_test.rb

```diff
@@ -19,17 +19,6 @@ class MinisterialRoleSearchIndexObserverTest < ActiveSupport::TestCase
     organisation.save
   end
 
-  test 'should not reindex all ministerial roles when an organisation is updated if observer is disabled' do
```

Comment:
I can't imagine why it would and there doesn't seem to be anywhere that it is happening from within the Whitehall code.